plugins/recommendations-plugin/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "recommendation-plugin",
```

Comment:
Could that be the same name everywhere ?
here `recommendation-plugin` but the folder is `recommendationS-plugin`
Also most of our plugins name start with `@eclipse-che/`

Reply:
it has been fixed

Reply:
@sunix I fixed it 13 days ago so could you review it again ?
You required changes for this bad convention, and I fixed it a couple of weeks ago